### PR TITLE
chore(vite): update vite version

### PR DESCRIPTION
**Pull request by mandarini**
Make sure to set the latest version of `vite` on `package.json`.

### CODE REVIEW

**Review by barbados-clemens — APPROVED** [review]
Does this also need bump vitest versions?

**Comment by mandarini** [comment]
> Does this also need bump vitest versions?

I don't think we need to, but I could add it sure!

**Comment by barbados-clemens** [comment]
> > Does this also need bump vitest versions?
> 
> I don't think we need to, but I could add it sure!

No worries, just wanted to make sure they weren't gonna get out of sync but looks like vitest dep is set to any minor so should be fine. 👍

File: packages/vite/src/executors/test/vitest.impl.ts [hunk 1]
```diff
@@ -93,14 +93,15 @@ async function getSettings(
 ) {
   const projectRoot = context.projectGraph.nodes[context.projectName].data.root;
   const offset = relative(workspaceRoot, context.cwd);
-  // if reportsDirectory is not provides vitest will remove all files in the project root
+  // if reportsDirectory is not provided vitest will remove all files in the project root
```
Comment (lukascivil) [inline packages/vite/src/executors/test/vitest.impl.ts:96]: The comma is missing, IMO.

`
// if reportsDirectory is not provided, vitest will remove all files in the project root
`

**Comment by pumano** [comment]
looks like PR is too old, 4.3.3 released also @angular-devkit/build-angular 16.0.0-rc2 depends indirectly from vite 4.3.1

**Review by barbados-clemens — APPROVED** [review]

**Review by FrozenPandaz — APPROVED** [review]

**Review by jaysoo — APPROVED** [review]

**Pull request merged**